kpops/cli/main.py

```diff
@@ -224,6 +224,8 @@ def generate(
                 '--steps'"
         )
 
+    return pipeline
```

Comment:
technically not needed anymore, was only used for testing. but perhaps it makes sense to return the generated pipeline anyways instead of only printing to stdout